internal/mode/static/state/graph/httproute.go

```diff
@@ -233,10 +233,8 @@ func buildRoute(
 		msg := allRulesErrs.ToAggregate().Error()
 
 		if atLeastOneValid {
-			// FIXME(pleshakov): Partial validity for HTTPRoute rules is not defined in the Gateway API spec yet.
-			// See https://github.com/nginxinc/nginx-gateway-fabric/issues/485
-			msg = "Some rules are invalid: " + msg
-			r.Conditions = append(r.Conditions, staticConds.NewTODO(msg))
+			msg = "Dropped Rule(s): " + msg
```

Comment:
Because dropped rule is part of the API (the prefix in the message), lets:
1. Move msg construction (`msg = "Dropped Rule(s): " + msg`) into NewRoutePartiallyInvalid
2. In NewRoutePartiallyInvalid, add a comment that notes that Dropped Rules(s) is part of the API, because it is not obvious -- there are no other cases in the Gateway API for that, something like:
```
// The message must start with "Dropped Rules(s)" according to the Gateway API spec
// See https://github.com/kubernetes-sigs/gateway-api/blob/37d81593e5a965ed76582dbc1a2f56bbd57c0622/apis/v1/shared_types.go#L408-L413
```
```